packages/theme/src/palette.ts

```diff
@@ -1,152 +1,164 @@
-    coinsCroSubdued: '#002D744D',
-    coinsDaiBold: '#F5AC37',
-    coinsDaiSubdued: '#F5AC374D',
-    coinsDashBold: '#1796E4',
-    coinsDashSubdued: '#1796E44D',
-    coinsDgbBold: '#3A75C9',
-    coinsDgbSubdued: '#3A75C94D',
-    coinsDogeBold: '#C8AF47',
-    coinsDogeSubdued: '#C8AF474D',
-    coinsEosBold: '#000000',
-    coinsEosSubdued: '#0000004D',
-    coinsErc20Bold: '#A6A8A9',
-    coinsErc20Subdued: '#A6A8A94D',
-    coinsEtcBold: '#60C67E',
-    coinsEtcSubdued: '#60C67E4D',
-    coinsEthBold: '#454A75',
-    coinsEthSubdued: '#454A754D',
-    coinsGntBold: '#8A92B2',
-    coinsGntSubdued: '#8A92B24D',
-    coinsLeoBold: '#171C1F',
-    coinsLeoSubdued: '#171C1F4D',
-    coinsLinkBold: '#2A5ADA',
-    coinsLinkSubdued: '#2A5ADA4D',
-    coinsLtcBold: '#A6A8A9',
-    coinsLtcSubdued: '#A6A8A94D',
-    coinsMaticBold: '#8247E5',
-    coinsMaticSubdued: '#8247E533',
-    coinsNameBold: '#2D78A4',
-    coinsNameSubdued: '#2D78A44D',
-    coinsNemBold: '#000000',
-    coinsNemSubdued: '#0000004D',
-    coinsOkbBold: '#000000',
-    coinsOkbSubdued: '#0000004D',
-    coinsShibBold: '#DD3935',
-    coinsShibSubdued: '#DD39354D',
-    coinsSolBold: '#000000',
-    coinsSolSubdued: '#0000004D',
-    coinsTnetBold: '#E75F5F',
-    coinsTnetSubdued: '#E75F5F4D',
-    coinsUniBold: '#ED2B88',
-    coinsUniSubdued: '#ED2B884D',
-    coinsUsdcBold: '#2775CA',
-    coinsUsdcSubdued: '#2775CA4D',
-    coinsUsdtBold: '#53AE94',
-    coinsUsdtSubdued: '#53AE944D',
-    coinsVtcBold: '#1B9065',
-    coinsVtcSubdued: '#1B90654D',
-    coinsWbtcBold: '#F29D40',
-    coinsWbtcSubdued: '#F29D404D',
-    coinsXlmBold: '#000000',
-    coinsXlmSubdued: '#0000004D',
-    coinsXmrBold: '#F4702A',
-    coinsXmrSubdued: '#F4702A4D',
-    coinsXrpBold: '#24292E',
-    coinsXrpSubdued: '#24292E4D',
-    coinsXtzBold: '#2C7DF7',
-    coinsXtzSubdued: '#2C7DF74D',
-    coinsZecBold: '#E0AB3C',
-    coinsZecSubdued: '#E0AB3C4D',
-    darkAccentBlue100: '#0E222A',
-    darkAccentBlue200: '#16262D',
-    darkAccentBlue300: '#1B2E37',
-    darkAccentBlue400: '#1E333E',
-    darkAccentBlue50: '#0B1C22',
-    darkAccentBlue500: '#233C48',
-    darkAccentBlue600: '#27789B',
-    darkAccentBlue700: '#2B85AB',
-    darkAccentBlue800: '#70BBDB',
-    darkAccentRed100: '#301717',
-    darkAccentRed200: '#351D1D',
-    darkAccentRed300: '#3F2222',
-    darkAccentRed400: '#452727',
-    darkAccentRed50: '#261212',
-    darkAccentRed500: '#522E2E',
-    darkAccentRed600: '#B65454',
-    darkAccentRed700: '#C15C5C',
-    darkAccentRed800: '#D89797',
-    darkAccentYellow100: '#29200A',
-    darkAccentYellow200: '#2E260F',
-    darkAccentYellow300: '#342B14',
-    darkAccentYellow400: '#3B3117',
-    darkAccentYellow50: '#231B06',
-    darkAccentYellow500: '#463A1B',
-    darkAccentYellow600: '#D2AC4B',
-    darkAccentYellow700: '#D8B864',
-    darkAccentYellow800: '#E2CA8D',
-    darkGray000: '#0F0F0F',
-    darkGray100: '#1D1D1D',
+    coinsADABold: '#3468D1',
```

Comment:
About these names, I'm waiting for next steps from David

Reply:
Duplicity to `packages/theme/src/coinsColors.ts`. We should delete one of these.